### PR TITLE
Add a way to encode a packet only once and send it to multiple clients

### LINKED ISSUE #713: Make it easy for a server to encode a packet once and send it to multiple clients
Basically use client.writeRaw + a convenient way to encode packets only once.

use case : https://github.com/PrismarineJS/flying-squid/blob/master/src/lib/plugins/world.js#L62

(@Karang idea)

would make server implementations faster

### PR DESCRIPTION
**Pull request by GroobleDierne**
Close #713

I assumed this will only be used with clients in "play" state.

### CODE REVIEW

**Review by u9g — APPROVED** [review]
lgtm

**Review by Karang — APPROVED** [review]
LGTM

**Comment by rom1504** [comment]
Could you add a test (in server.js in test folder) ?
For example send chat to 2 clients and make sure they receive it.
This is a pretty fundamental feature so I think it deserves testing

**Comment by GroobleDierne** [comment]
Yes I'll do it when I have the time

**Comment by GroobleDierne** [comment]
Ready 👍

File: test/serverTest.js [hunk 1]
```diff
@@ -370,5 +371,66 @@ mc.supportedVersions.forEach(function (supportedVersion, i) {
...
+        player1.on('chat', (packet) => {
+          assert.strictEqual(packet.message, '{"text":"A message from the server."}')
+          player1.end()
+        })
+        player2.on('chat', (packet) => {
+          assert.strictEqual(packet.message, '{"text":"A message from the server."}')
+          player2.end()
+        })
+        
+        server.writeToClients(Object.values(server.clients), 'chat', { message: JSON.stringify({ text: 'A message from the server.' }), position: 1, sender: '00000000-0000-0000-0000-000000000000' })
```
Comment (rom1504) [inline test/serverTest.js:430]: you need to wait for client to receive the chat message

**Review by rom1504 — CHANGES_REQUESTED** [review]

**Comment by GroobleDierne** [comment]
Done

**Pull request merged**